Source/Core/VideoCommon/HiresTextures.h

```diff
@@ -14,11 +14,15 @@
 
 enum class TextureFormat;
 
+std::set<std::string> GetTextureDirectoriesWithGameid(const std::string& root_directory,
```

Comment:
```suggestion
std::set<std::string> GetTextureDirectoriesWithGameId(const std::string& root_directory,
```

(sorry for nitpicking)

Reply:
No problem at all.  Had to look for a while to see what was different :)